### PR TITLE
chore(CODEOWNERS): add CODEOWNERS to repo

### PR DESCRIPTION
**Pull request by princhcanal**
## Description

- adds `myern-plus-princh` as CODEOWNERS to all files

## Checklist

_First, create a draft pull request. Then mark the PR as ready for review when all necessary checkbox items has been completed_

- [x] Reviewed own code
- [x] Commented on code that is hard to understand
- [ ] Implemented tests for the feature/bugfix
- [x] All GitHub status checks pass
- [x] The frontend and backend PR previews have been deployed

**Backend only**

- [ ] Generated the client api if there are new or deleted endpoints and/or DTOs

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @citu-cpe/myern-plus-princh
```
Comment (princhcanal) [inline .github/CODEOWNERS:1]: this file lets github know which team should be asked to review pull requests for this repository. `*` means that changes to any file should be reviewed by the whole team

**Comment by beefysalad** [comment]
HMMMM

**Comment by princhcanal** [comment]
> HMMMM

walay approval do? 💔 unsa paman ako e change 😢

File: .github/CODEOWNERS (hunk 1 above)
Comment (beefysalad) [reply .github/CODEOWNERS:1]: LGTM

**Review by beefysalad — APPROVED** [review]

**Comment by beefysalad** [comment]
lgtm!

**Comment by noice-noise** [comment]
'Tis amazing ser! 10/10

**Review by noice-noise — APPROVED** [review]

**Pull request merged**